shared/js/ui/templates/shared/hero.es6.js

```diff
@@ -3,22 +3,26 @@ const bel = require('bel')
 module.exports = function (ops) {
     const slidingSubviewClass = ops.showClose ? 'js-sliding-subview-close' : ''
     return bel`<div class="hero border--bottom text--center ${slidingSubviewClass} silver-bg">
-    ${renderOpenOrCloseButton(ops.showClose)}
     <div class="hero__icon hero__icon--${ops.status}">
     </div>
     <h1 class="hero__title">
         ${ops.title}
     </h1>
-    <h2 class="hero__subtitle">
+    <h2 class="hero__subtitle" aria-label="${ops.subtitleLabel ? ops.subtitleLabel : ops.subtitle}">
```

Comment:
If the subtitleLabel passed is falsy or an empty string, we just use the subtitle itself as a label.

In the vast majority of cases, the subtitles are descriptive enough (e.g. "Good privacy practices"). Any other situations are covered in the `rating-hero` template.